### ISSUE TITLE
ERROR 2013 (HY000): Lost connection to MySQL server during query

### ISSUE THREAD

**Issue by bguruprasad** [y-ken/mysql-udf-base64#2]
Hi, 
This function works pretty fine when used on small dataset, < 5 records. When used on large dataset it gives `ERROR 2013 (HY000): Lost connection to MySQL server during query` I want to use it on more then 1000 rows, how to archive the same? It is even possible or not???

**Comment by y-ken** [comment]
Hi. 
I thought some types of record cause the error. it might not be dataset size.
Would you try to find the record which is the trigger.
Anyway, It seems this UDF needs some error handling.

**Comment by bguruprasad** [comment]
@y-ken, I have inserted same records multiple times to test this UDF, but still it is giving same error.

**Comment by y-ken** [comment]
Hi

If you're using MySQL 5.6.1 or higher you can use the TO_BASE64  / FROM_BASE64() function.
https://dev.mysql.com/doc/refman/5.6/en/string-functions.html#function_from-base64

```sql
-- note: This function was added in MySQL 5.6.1.
mysql> SELECT TO_BASE64('base64 encoded string');
-> YmFzZTY0IGVuY29kZWQgc3RyaW5n
```

**Comment by bguruprasad** [comment]
Yes, I'm aware of this new string function available in MySQL v5.6 above, I have MySQL v5.5 in production and want to use this UDF for my propose. :)
